cannon/mipsevm/tests/evm_common_test.go

```diff
@@ -162,6 +162,72 @@ func TestEVMSingleStep(t *testing.T) {
 	}
 }
 
+func TestEVMSingleStep_Add(t *testing.T) {
+	var tracer *tracing.Hooks
+
+	versions := GetMipsVersionTestCases(t)
+	cases := []struct {
+		name         string
+		pc           uint32
+		nextPC       uint32
```

Comment:
nit: Can hardcode `pc`, `nextPC`, and `expectedNextPC` since they do not change across tests.